test/cli/proj-inline-suppress-unusedFunction/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.4)
```

Comment:
It seems to me this CMakeLists.txt is redundant. A hardcoded compile_commands.json is nice in the test because then we know exactly what that contains.

Reply:
I agree. I will be removing it. My original plan was to actually use CMake to create the compile_commands.json file.